x-pack/plugins/security_solution/public/common/components/header_actions/actions.tsx

```diff
@@ -212,15 +209,11 @@ const ActionsComponent: React.FC<ActionProps> = ({
     }
     onEventDetailsPanelOpened();
   }, [activeStep, incrementStep, isTourAnchor, isTourShown, onEventDetailsPanelOpened]);
-  const showExpandEvent = useMemo(
-    () => !unifiedComponentsInTimelineEnabled || isEventViewer,
```

Comment:
We can probably get rid of `isEventViewer` all together as I think it's a tech debt artifact and is set to `false` everywhere I checked. Not going to do that here though to prevent breaking anything else